Remotion/ObjectBinding/Web.ClientScript/Scripts/typings/jquery/jquery.d.ts

```diff
@@ -2331,7 +2331,7 @@ interface JQuery {
      * @param eventType A string containing one or more DOM event types, such as "click" or "submit," or custom event names.
      * @param handler A function to execute each time the event is triggered.
      */
-    bind(eventType: string, handler: (eventObject: JQueryEventObject) => any): JQuery;
+    bind(eventType: string, handler: (eventObject: JQueryEventObject, ...eventHandlerArgs: any[]) => any): JQuery;
```

Comment:
@a-ctor I chased the bugs for a while and it finally seems that this was it. It doesn't say anywhere in the official docs that this is supposed to be the signature but it's apparently hidden away in the examples of the `bind` [docs](https://api.jquery.com/bind/):

```js
$( "p" ).bind( "myCustomEvent", function( e, myName, myValue ) {
  $( this ).text( myName + ", hi there!" );
  $( "span" )
    .stop()
    .css( "opacity", 1 )
    .text( "myName = " + myName )
    .fadeIn( 30 )
    .fadeOut( 1000 );
  });
```

I deleted the second argument anywhere where it was used since it didn't match the signature provided in this file and it looks like that's what caused the builds to fail